### DISCUSSION TITLE
Future request: Apple Pencil double tap

### DISCUSSION THREAD

**Discussion by senseofeternity** [zsviczian/obsidian-excalidraw-plugin#981] [Ideas]
Hello, many thanks for the plugin. 
When using an iPad Pro with the Apple Pencil (2nd generation), the double tap on pencil to switch between pen and eraser is a very familiar gesture, to which I am very get used. It works in all apps, like apple notes, notability, good notes, etc. 
Could you please add this feature? It would be really helpful. Thanks.

**Comment by DCZare** [comment]
Please add this feature!

**Comment by julienfaure** [comment]
+1
Double tap on screen would trigger previous button.

**Comment by Josh-Piper** [comment]
Would love this
